### PR TITLE
Extend Croatia with crop mapping

### PR DESCRIPTION
**Pull request by ivorbosloper**

### CODE REVIEW

**Comment by m-mohr** [comment]
A changelog please :-)

File: fiboa_cli/datasets/hr.py [hunk 1]
```diff
@@ -141,12 +141,47 @@
...
+321;Krški pašnjak;Karst pasture
+410;Vinograd;Vineyard
+411;Iskrčeni vinograd;Cleared vineyard
+421;Maslinik;Olive grove
+422;Voćnjak	Poljoprivredno;Orchard
+430;Kulture kratke ophodnje;Short tour cultures
+450;Rasadnik;Nursery
+490;Mješoviti višegodišnji nasadi;Mixed perennial plantations
+900;Ostale vrste uporabe zemljišta;Other types of land use
+910;Privremeno neodržavana parcela;Temporarily unmaintained plot"""
```
Comment (m-mohr) [inline fiboa_cli/datasets/hr.py:172]: I'm wondering whether we should store these in separate files...

Comment (ivorbosloper) [reply fiboa_cli/datasets/hr.py]: We could store it as a separate file. I made a proposal

File: fiboa_cli/datasets/commons/data.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from csv import DictReader
+from os.path import dirname, join
+
+
+def read_data_csv(name, **kwargs):
+    return list(DictReader(open(join(dirname(dirname(__file__)), "data-files", name), 'r'), **kwargs))
```
Comment (m-mohr) [inline fiboa_cli/datasets/commons/data.py:6]:
```suggestion
    path = join(dirname(dirname(__file__)), "data-files", name)
    with open(path, 'r') as f:
        return list(DictReader(f, **kwargs))
```

**Review by m-mohr — APPROVED** [review]
A small code suggestion, otherwise LGTM

File: fiboa_cli/datasets/hr.py (hunk 1 above)
Comment (m-mohr) [reply fiboa_cli/datasets/hr.py]: Thanks

**Pull request merged**